projects/aca-content/folder-rules/src/manage-rules/manage-rules.smart-component.spec.ts

```diff
@@ -183,9 +208,9 @@ describe('ManageRulesSmartComponent', () => {
 
     expect(component).toBeTruthy();
 
-    const acaGenericError = debugElement.query(By.css('aca-generic-error'));
-    const rules = debugElement.query(By.css('.aca-rule-list-item'));
-    const ruleDetails = debugElement.query(By.css('aca-rule-details'));
+    const acaGenericError = unitTestingUtils.getByCSS('aca-generic-error');
+    const rules = unitTestingUtils.getByCSS('.aca-rule-list-item');
+    const ruleDetails = unitTestingUtils.getByCSS('aca-rule-details');
```

Comment:
aca-rule-details is selector of component so please use getByDirective function instead to get that element